File: 1-50/1-9/6/PE006.java
public class PE006 {
    private static double sumSquare() {
        double sum = 0;
        for (int i = 1; i < 101; i++) {
            sum += i * i;
        }

        return sum;
    }

    private static double squareSum() {
        double sum = (100.0 / 2) * (1 + 100); // Arithmetic Sequence
        return Math.pow(sum, 2);
    }

    public static void main(String[] args) {
        System.out.println((int) (squareSum() - sumSquare()));
    }
}

File: 1-50/1-9/6/PE006.py
max = 100

def sumSquare():
    sum = 0
    for i in range(1, max + 1):
        sum += i ** 2

    return sum

def squareSum():
    sum = int(max / 2 * (1 + max)) # Arithmetic Sequence
    return sum ** 2

print(squareSum() - sumSquare())
